airflow/providers/amazon/aws/executors/batch/__init__.py

```diff
@@ -14,3 +14,12 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
+from __future__ import annotations  # Added by precommit hooks
+
+# from airflow.providers.amazon.aws.executors.batch.batch_executor import AwsBatchExecutor
+from airflow.providers.amazon.aws.executors.batch import batch_executor
+
+# precommit hooks (rust as of the time of commit) throws F401 - "Module imported but unused"
+# One of the solutions it suggests is using a "redundant alias". This is used below instead of doing a #no-qa
+# type ignore of the issue.
+AwsBatchExecutor = batch_executor.AwsBatchExecutor
```

Comment:
```suggestion
from __future__ import annotations

__all__ = ["AwsBatchExecutor"]

from airflow.providers.amazon.aws.executors.batch.batch_executor import AwsBatchExecutor
```

`__all__` will make pre-commit and IDEs happy without hacks

Reply:
To me using `__all__` is just as much of a hack, since we wouldn't specify that anyway. I'm not sure one is better than the other.

Reply:
Moved to using __all__